docs/concepts/policy/pod-security-policy.md

```diff
@@ -125,141 +399,153 @@ to the volume sources that are defined when creating a volume:
+provided. Setting `AllowPrivilegeEscalation=false` is strongly recommended with
+this strategy.
+- *RunAsAny* - No default provided. Allows any `runAsUser` to be specified.
 
-```shell
-$ kubectl get psp
-NAME        PRIV   CAPS  SELINUX   RUNASUSER         FSGROUP   SUPGROUP  READONLYROOTFS  VOLUMES
-permissive  false  []    RunAsAny  RunAsAny          RunAsAny  RunAsAny  false           [*]
-privileged  true   []    RunAsAny  RunAsAny          RunAsAny  RunAsAny  false           [*]
-restricted  false  []    RunAsAny  MustRunAsNonRoot  RunAsAny  RunAsAny  false           [emptyDir secret downwardAPI configMap persistentVolumeClaim projected]
-```
+**SupplementalGroups** - Controls which group IDs containers add.
 
-## Editing a Pod Security Policy
+- *MustRunAs* - Requires at least one range to be specified. Uses the
+minimum value of the first range as the default. Validates against all ranges.
+- *RunAsAny* - No default provided. Allows any `supplementalGroups` to be
+specified.
 
-To modify policy interactively, use `kubectl edit`:
+### Privilege Escalation
+
+These options control the `AllowPrivilegeEscalation` container option. This bool
+directly controls whether the
+[`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt)
+flag gets set on the container process. This flag will prevent `setuid` binaries
+from changing the effective user ID, and prevent files from enabling extra
+capabilities (e.g. it will prevent the use of the `ping` tool). This behavior is
+required to effectively enforce `MustRunAsNonRoot`.
+
+It defaults to `nil`. The default behavior of `nil` allows privilege escalation
+so as to not break setuid binaries. Setting it to `false` ensures that no child
+process of a container can gain more privileges than its parent.
+
+**AllowPrivilegeEscalation** - Gates whether or not a user is allowed to set the
+security context of a container to `allowPrivilegeEscalation=true`. This
+defaults to allowed.
+
+**DefaultAllowPrivilegeEscalation** - Sets the default for the
+`AllowPrivilegeEscalation` option. The default behavior is to allow privilege
+escalation so as to not break setuid binaries.
+
+### Capabilities
+
+Linux capabilities provide a finer grained breakdown of the privileges
+traditionally associated with the superuser. Some of these capabilities can be
+used to escalate privileges or for container breakout, and may be restricted by
+the PodSecurityPolicy. For more details on Linux capabilities, see
+[Capabilities(7)](http://man7.org/linux/man-pages/man7/capabilities.7.html).
+
+The following fields take a list of capabilities, specified as the capability
+name in ALL_CAPS without the `CAP_` prefix.
+
+**AllowedCapabilities** - Provides a whitelist of capabilities that may be added
+to a container. The default set of capabilities are implicitly allowed. The
+empty set means that no additional capabilities may be added beyond the default
+set.
```

Comment:
I think that we also could mention about `*` here.

Reply:
done.